### ISSUE TITLE
Search no longer works after 0.33.1 upgrade

### ISSUE THREAD

**Issue by CoconutTherapy** [metabase/metabase#10806]
**Describe the bug**
Search loads indefinitely and does not return results.

**Logs**
```javascript
Uncaught TypeError: a is not a function
    at app-main.bundle.js?53f423daaafc18f862bc:5
    at Array.map (<anonymous>)
    at app-main.bundle.js?53f423daaafc18f862bc:5
    at renderChildren (app-main.bundle.js?53f423daaafc18f862bc:5)
    at e (app-main.bundle.js?53f423daaafc18f862bc:5)
    at t.value (app-main.bundle.js?53f423daaafc18f862bc:5)
    at t.value (app-main.bundle.js?53f423daaafc18f862bc:5)
    at u._renderValidatedComponentWithoutOwnerOrContext (vendor.bundle.js?53f423daaafc18f862bc:1)
    at u._renderValidatedComponent (vendor.bundle.js?53f423daaafc18f862bc:1)
    at u._updateRenderedComponent (vendor.bundle.js?53f423daaafc18f862bc:1)
```

**To Reproduce**
Upgrade to 0.33.1
Make a search

**Expected behavior**
Search should return results

**Screenshots**
![image](https://user-images.githubusercontent.com/29283297/64333558-0fb5c600-cfd7-11e9-8fa4-22936eab8e1f.png)

**Information about your Metabase Installation:**
- Your browser and the version: Chrome 76.0.3809.132
- Your operating system: Windows 10
- Metabase version: 0.33.1
- Metabase hosting environment: Elastic Beanstalk
- Metabase internal database: Postgres

**Severity**
Pretty severe, we can't search anymore

**Comment by kritop** [comment]
same here, just updated to 0.33.1 from 0.33.0 (Heroku with Postgres)

**Comment by paulrosenzweig** [comment]
Fixed by #10811
